src/test/isPhone.spec.ts

```diff
@@ -0,0 +1,123 @@
+import { expect } from 'chai'
+import { IsPhoneNumber } from '../main/isPhone'
+import { validateSync } from 'class-validator'
+
+class PhoneNumberTest {
+  @IsPhoneNumber()
+  value: any
+
+  constructor (value: any) {
+    this.value = value
+  }
+}
+
+/**
```

Comment:
That comments is not valid for phone validator I guess. Mobile phone validator is aligned with GOV.UK Notify validation rules but phone number validator is not.